core/src/App.html

```diff
@@ -545,9 +547,37 @@
         });
       },
       onResize() {
-        if (window.innerWidth >= CSS_BREAKPOINTS.desktopMinWidth) {
+        const simpleDesktopToMobile = () => {
+          if (this.get().responsiveNavSetting == 'simple') {
+            if (
+              window.innerWidth < CSS_BREAKPOINTS.desktopMinWidth &&
+              this.get().previousBreakpoint >= CSS_BREAKPOINTS.desktopMinWidth
+            ) {
+              this.closeLeftNav();
+            }
+          }
+        };
+
+        const simpleIsNotSameBreakpoint = () => {
+          if (this.get().responsiveNavSetting == 'simple') {
+            if (
+              this.get().previousBreakpoint >= CSS_BREAKPOINTS.desktopMinWidth
+            ) {
+              return false;
+            }
+          }
+          return true;
+        };
+
+        if (
+          window.innerWidth >= CSS_BREAKPOINTS.desktopMinWidth &&
+          simpleIsNotSameBreakpoint()
+        ) {
           this.closeLeftNav();
         }
+
+        simpleDesktopToMobile();
+        this.get().previousBreakpoint = window.innerWidth;
```

Comment:
Please use better ```this.set({ previousBreakpoint: window.innerWidth });```. Otherwise we are doing a mutation of the component state. It is better to let Svelte do the state updates itself, by just telling Svelte what we want to update with `this.set`.